packages/client/src/components/questions.jsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable max-len */
+import React, { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const options1 = [
```

Comment:
Should this be in the database?

Reply:
it probably could be stored in the database, but it's hard coded for this assignment